content/docs/command-reference/exp/remove.md

```diff
@@ -5,8 +5,8 @@ Delete specific `dvc experiments` from the <abbr>project</abbr>.
 ## Synopsis
 
 ```usage
-usage: dvc exp remove [-h] [-q | -v] [--queue]
-                      [experiment [experiment ...]]
+usage: dvc experiments remove [-h] [-q | -v] [--queue | -A]
+                      [<experiment> [<experiment> ...]]
```

Comment:
Why change these parts? 🙂 breaks aligning too
```suggestion
usage: dvc exp remove [-h] [-q | -v] [--queue | -A]
                      [experiment [experiment ...]]
```

Reply:
I copied the usage line from 
```
dvc exp remove --help
```
I guess we need to fix it there.

Reply:
Yeah I imagined that was it, no worries. We don't need to have the exact same thing in both places. (May be hard to customize in Python argparse). Ok I'm committing this too...

Reply:
> I guess we need to fix it there.

p.s. @karajan1001 well if it shows `<experiment>` as the argument name I think that should be changed in the core repo, yes. We only use `<name>` for option argument metavars I think.

Reply:
This one matter left here though @karajan1001 ☝️